.github/workflows/canary-integration-test.yml

```diff
@@ -1614,7 +1614,7 @@ jobs:
           name: ${{ github.job }}-${{ matrix.ceph-image }}
           additional-namespace: kube-system
 
-  csi-hostnetwork-disabled:
+  multus-cluster-network-without-holder:
```

Comment:
Could we also rename the test above to be `multus-cluster-network-with-holder` so it's clear in the outputs which is which?